hotel-reservation-system/src/components/Hotel/ServiceTable.js

```diff
@@ -107,6 +118,26 @@ export default function ServiceTable({ hotelId }) {
   function handleCloseDeleteDialog() {
     setOpenDeleteDialog(false);
   }
+  function increaseQuantity(service) {
+    var newService = {
```

Comment:
Please never use 'var'. Use 'const' or 'let' ('let' if you are going to reassign the variable only) instead.

Reply:
all var types replaced by let